iframe/style.css

```diff
@@ -2,9 +2,10 @@ body {
   height: 100%;
   margin: 0;
   padding: 0;
+  background: rgba(0, 0, 0, .56);
   font-family: 'Open Sans', sans-serif;
   color: white;
-  overflow: hidden;
+  overflow: scroll;
```

Comment:
I *think* this might add scrollbars to both x and y dimensions, depending on the browser. `overflow-y: scroll`.